README.md

```diff
@@ -88,3 +88,44 @@ $authorizationChecker = $container->get(/* ... */);
 $authorizationChecker->isGranted(array('IS_OWNER'), $germinal); // true
 $authorizationChecker->isGranted(array('IS_OWNER'), $miserables); // false
 ```
+
+##Security from routing
+
+You can specify security constraints directly from your routing. You would provide a role or an array of roles with the `roles` parameter. If you specify an array, it will be passed *as is* to the authorization checker, and that means the [authorization strategy](http://symfony.com/doc/current/cookbook/security/voters.html#changing-the-access-decision-strategy) depends on your configuration of the security component.
```

Comment:
ou can specify security constraints directly from your routing by providing a role or an array of roles with the `roles` parameter.